game/scripts/npc/abilities/queenofpain_blink.txt

```diff
@@ -33,7 +33,7 @@
       "shard_damage" //OAA
       {
         "RequiresShard"				"1"
-        "value"						"250"
+        "value"						"100 150 200 250 500 750"
```

Comment:
i feel like thats too much dmg at oaa levels. Like this is a buff for QOP outside of the stupid shard modifier, which shouldn't influence balancing eitherway.